src/validators/MultiFactor.sol

```diff
@@ -0,0 +1,83 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.19;
+
+import "@openzeppelin/contracts/utils/cryptography/MerkleProof.sol";
+import "modulekit/common/erc4337/Helpers.sol";
+import "modulekit/modulekit/ValidatorBase.sol";
+import "modulekit/modulekit/lib/ValidatorSelectionLib.sol";
+
+import "sentinellist/src/SentinelList.sol";
+
+import "forge-std/console2.sol";
+
+contract MultiFactor is ValidatorBase {
+    using ValidatorSelectionLib for UserOperation;
+    using SentinelListLib for SentinelListLib.SentinelList;
+
+    mapping(address account => SentinelListLib.SentinelList) private subValidators;
```

Comment:
This wont pass the 4337 validation specs fyi

Reply:
Good point. Id say we hold off with the merge for now. let me find a better solution for this that complies with 4337